lib/MediaConnection.d.ts

```diff
@@ -0,0 +1,21 @@
+import * as EventEmitter from "events";
+
+type MediaConnectionType = 'NodeMediaConnection' | 'BrowserMediaConnection';
+
+export interface MediaConnection extends EventEmitter {
+    receiveSdpOffer(options: any): Promise<RTCSessionDescriptionInit>
+
+    receiveSdpAnswer(options: any): Promise<RTCSessionDescriptionInit>
+
+    setLocalSdp(sdp: RTCSessionDescriptionInit): Promise<void>
+
+    setRemoteSdp(sdp: RTCSessionDescriptionInit): Promise<void>
+
+    sendInbandDTMF(tones: number, duration: number, interToneGap: number): void
```

Comment:
I'd rename it to `sendDTMF` so we don't expose how DTMFs are sent. This is just an interface.

Reply:
Why i should remove the integrated SIP-DTMF? Actually only the 'inband' generation has to be implemented by the implementation. 
Hm, i can also remove the entire part and include the SIP part in the default implementation

Reply:
Not about SIP at all, but I may implement a `MediaConnection` class that uses whatever technique (does not even need to be standard) to send a DTMF, that's why I suggest renaming it to `sendDTMF`, but nothing to do with SIP.